en/micro-integrator/docs/references/connectors/servicenow-connector/servicenow-connector-example.md

```diff
@@ -0,0 +1,250 @@
+          <number>34</number>
+          <shortDescription>{$ctx:shortDescription}</shortDescription>
+          <active>true</active>
+          <approval>owner</approval>
+          <category>inquiry</category>
+          <contactType>{$ctx:contactType}</contactType>
+      </servicenow.postRecord>
+      <property expression="json-eval($.result.sys_id)" name="sysId" scope="default" type="STRING"/>
+      </sequence>
+    ```
+3. Create the ReadRecord sequence as below. 
+  ```
+    <?xml version="1.0" encoding="UTF-8"?>
+    <sequence name="ReadRecord" trace="disable" xmlns="http://ws.apache.org/ns/synapse">
+        <servicenow.init>
+            <serviceNowInstanceURL>https://dev55707.service-now.com</serviceNowInstanceURL>
+            <username>admin</username>
+            <password>Diazo123@</password>
+        </servicenow.init>
+        <servicenow.getRecordById>
+            <sysId>{$ctx:sysId}</sysId>
+            <tableName>incident</tableName>
+        </servicenow.getRecordById>
+    </sequence>
+  ```
+  4. Now right click on the created ESB Solution Project and select, -> **New** -> **Rest API** to create the REST API. 
+    <img src="/assets/img/connectors/adding-an-api.png" title="Adding a Rest API" width="800" alt="Adding a Rest API"/>
+  
+  5. Provide the API name as ServiceNowAPI and the API context as `/servicenow`. You can go to the source view of the xml configuration file of the API and copy the following configuration. 
+    ```
+      <?xml version="1.0" encoding="UTF-8"?>
+      <api context="/servicenow" name="ServiceNowAPI" xmlns="http://ws.apache.org/ns/synapse">
+          <resource methods="POST" uri-template="/postRecord">
+              <inSequence>
+                  <property expression="json-eval($.shortDescription)" name="shortDescription" scope="default" type="STRING"/>
+                  <property expression="json-eval($.contactType)" name="contactType" scope="default" type="STRING"/>
+                  <sequence key="PostRecord"/>
+                  <respond/>
+              </inSequence>
+              <outSequence/>
+              <faultSequence/>
+          </resource>
+          <resource methods="POST" uri-template="/readRecord">
+              <inSequence>
+                <property expression="json-eval($.sysId)" name="sysId" scope="default" type="STRING"/>
+                  <sequence key="ReadRecord"/>
+                  <respond/>
+              </inSequence>
+              <outSequence/>
+              <faultSequence/>
+          </resource>
+      </api>
+
+    ```
+
+{!references/connectors/exporting-artifacts.md!}
+
+
+## Deployment
+
+Follow these steps to deploy the exported CApp in the Enterprise Integrator Runtime. 
+
+{!references/connectors/deploy-capp.md!}
+
+## Testing
+
+### Post Record Operation
+
+1. Create a file called data.json with the following payload. 
+    ```
+    {
+        "shortDescription":"Incident type: L2",
+        "contacttype":"email"
+    }
+    ```
```

Comment:
Reader will wonder what should come as `contacttype `. Shall we point to the reference doc here?

Reply:
Added